apps/web/pages/project/[id].tsx

```diff
@@ -24,14 +24,22 @@ export const getStaticProps: GetStaticProps = async ({ params, locale }) => {
 
     return addApolloState(client, {
       props: {
-        ...(await serverSideTranslations(locale, ['common', 'project'])),
+        ...(await serverSideTranslations(locale, [
+          'common',
+          'project',
+          'report-modal',
```

Comment:
add the translations in project.json instead of creating another file.

Reply:
and add it in every language

Reply:
>...............<